gdk/telemetry/metric.py

```diff
@@ -0,0 +1,32 @@
+import logging
+import time
+from enum import Enum
+
+from gdk._version import __version__
+
+logger = logging.getLogger(__name__)
+
+
+MetricType = Enum('MetricTypes', ["INSTALLED", "BUILD", "PUBLISH", "TEMPLATE", "PING"])
```

Comment:
Can you change this to COMPONENT_BUILD, COMPONENT_PUBLISH?

Reply:
We chatted offline. I will do it as a follow up